modules/book.js

```diff
@@ -0,0 +1,15 @@
+export default class Book {
+  constructor() {
+    this.bookArray = localStorage.getItem('bookstore') ? JSON.parse(localStorage.getItem('bookstore')) : [];
+  }
+
+  addNewBook(book) {
+    this.bookArray.push(book);
+    localStorage.setItem('bookstore', JSON.stringify(this.bookArray));
+  }
+
+  removeBook(id) {
+    this.bookArray = this.bookArray.filter((book) => book.id !== Number(id));
+    localStorage.setItem('bookstore', JSON.stringify(this.bookArray));
+  }
```

Comment:
- [ ] according to the requirements you should use ES6 syntax for this milestone, which means that all functions you created should be arrow functions

Reply:
Hi @mikethreels,
the javascript class replaces the function in this case, so `addBook and `removeBook` are just methods inside the class